tests/app/service/test_rest.py

```diff
@@ -1438,40 +1435,41 @@ def test_get_detailed_services_for_date_range(
     start_date_delta,
     end_date_delta,
 ):
-    from app.service.rest import get_detailed_services
+    try:
```

Comment:
Only the asserts should be in the "try" (PEP8).

Reply:
pytest doesn't even need try/except blocks because pytest knows how to handle exceptions. does the test run without the try block?

Reply:
Updated.